### PR TITLE
Sanitize SQL so the build process can continue

### PR DESCRIPTION
**Pull request by JonBendtsen**
The build process in my other pull requests failed so I decided to fix it.

### CODE REVIEW

File: htdocs/admin/system/dbtable.php [hunk 1]
```diff
@@ -52,10 +52,10 @@
 		while ($i < $num) {
 			$row = $db->fetch_row($resql);
 			if ($row[0] == $field) {
-				$sql = 'ALTER TABLE '.$db->sanitize($table).' MODIFY '.$db->sanitize($row[0]).' '.$row[1].' CHARACTER SET utf8';		// We must not sanitize the $row[1]
+				$sql = 'ALTER TABLE '.$db->sanitize($table).' MODIFY '.$db->sanitize($row[0]).' '.$db->sanitize($row[1]).' CHARACTER SET utf8';		// We must not sanitize the $row[1]
```
Comment (frederic34) [inline htdocs/admin/system/dbtable.php:55]: @JonBendtsen have you seen the comment `// We must not sanitize the $row[1]`

Comment (JonBendtsen) [reply htdocs/admin/system/dbtable.php:55]: > @JonBendtsen have you seen the comment `// We must not sanitize the $row[1]`

No I did not see that, I just wanted my other pull requests to build. I guess I can just close this pull request then.

Comment (frederic34) [reply htdocs/admin/system/dbtable.php:55]: let's wait and see what the coder wanted to do

Comment (frederic34) [reply htdocs/admin/system/dbtable.php:55]: https://github.com/Dolibarr/dolibarr/issues/27998

**Pull request closed without merging**